esprima.js

```diff
@@ -3955,6 +3956,12 @@
             }
         }
 
+        if (matchKeyword('let') || matchContextualKeyword('let')) {
```

Comment:
> Consider adding a function called matchLet as this logic is used in several places.

I thought about this, but I think it's only used twice and adding an extra mental note (assuming you're already familiar with `matchKeyword` and `matchContextualKeyword`) does not seem to be worth the hassle.